src/lib/prompts.ts

```diff
@@ -33,14 +41,64 @@ export const privateKeyPrompt = async (): Promise<string> => {
   return data;
 };
 
-export const tenderlyKeyPrompt = async (): Promise<string> => {
-  const { data } = await inquirer.prompt({
-    type: 'password',
-    name: 'data',
-    message: 'Please enter a Tenderly key:',
+export const tenderlyPrompt = async (): Promise<TenderlySettings> => {
+  const { tenderlyUsername, tenderlyProject, tenderlyKey } =
+    await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'tenderlyUsername',
+        message: prompts.TENDERLY_USERNAME,
+        validate: (input: string) => !!input || 'Username cannot be empty',
+      },
+      {
+        type: 'input',
+        name: 'tenderlyProject',
+        message: prompts.TENDERLY_PROJECT,
+        validate: (input: string) => !!input || 'Project name cannot be empty',
+      },
+      {
+        type: 'password',
+        name: 'tenderlyKey',
+        message: prompts.TENDERLY_KEY,
+        validate: (input: string) => {
+          try {
+            tenderlyKeySchema.parse(input);
+            return true;
+          } catch (error) {
+            if (error instanceof ZodError) {
+              return error.errors[0].message;
+            }
+            return 'Invalid input.';
```

Comment:
Let's return the `error` here itself so that's it's easier to debug for devs later